doc/doxyfile.in

```diff
@@ -2046,7 +2046,7 @@ TAGFILES               =
 # tag file that is based on the input files it reads. See section "Linking to
 # external documentation" for more information about the usage of tag files.
 
-GENERATE_TAGFILE       =
+GENERATE_TAGFILE       = @DOXYGEN_OUTPUT_DIR@/html/SFML.tag
```

Comment:
I wouldn't put this file in the `html` sub-directory as it's unrelated to the HTML generated files. One could build the doc using other output format instead. Of course, the other change should be adapted.

Reply:
Well, I think the tagfile mechanism is only used to link against HTML documentations, but I might have misunderstood that ? Doxygen's page :
https://www.stack.nl/~dimitri/doxygen/manual/external.html

If that is the case I personnaly think it's natural to have it in the HTML sub-dir.

Reply:
The example on the site you linked, doesn't add it inside the html directory. As such I wouldn't place it there either.

Reply:
Okay, I get it.
It can be installed in the doc directory (parent of html), would this make more sense ? This way it wouldn't be inside the html directory but is still installed in an already standard place.

Reply:
Oh, and as I am not used to work with pull requests, reviews and everything : should I commit the changes corresponding to this request or what ?

Reply:
It's your PR, so yes, you should provide the change. Additionally, if you could squash your commits, that would be great as well, otherwise I'll just do it when merging. :wink:

Reply:
> It can be installed in the doc directory (parent of html), would this make more sense ? 

Yes. :)